backend/api-service/src/main/resources/application.yml

```diff
@@ -0,0 +1,11 @@
+spring:
+  datasource:
+    url: jdbc:mysql://localhost:3306/boutique_hotel
+    username: SET_ME
+    password: SET_ME
+    driver-class-name: com.mysql.cj.jdbc.Driver
+  jpa:
+    database-platform: org.hibernate.dialect.MySQL8Dialect
```

Comment:
I would remove it, as:
2024-05-31T15:11:57.125+02:00  WARN 17796 --- [api-service] [           main] org.hibernate.orm.deprecation            : HHH90000025: MySQL8Dialect does not need to be specified explicitly using 'hibernate.dialect' (remove the property setting and it will be selected by default)
2024-05-31T15:11:57.125+02:00  WARN 17796 --- [api-service] [           main] org.hibernate.orm.deprecation            : HHH90000026: MySQL8Dialect has been deprecated; use org.hibernate.dialect.MySQLDialect instead